.github/actions/sign-attestation/bin/sign-attestation.js

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
```

Comment:
this file is only needed if we use a composite Action and don't call `dist/index.js` directly

Reply:
Ahh gotcha. I'll update to remove this. This entire action might get superseded anyway by an action that can take in the SLSA_OUTPUT_ARTIFACTS and loop over signing code.